src/main/java/net/sf/jabref/gui/BasePanel.java

```diff
@@ -2142,11 +2142,15 @@ public void action() {
 
     private class PrintPreviewAction implements BaseAction {
 
-         @Override
-         public void action() throws Throwable {
+        @Override
+        public void action() throws Throwable {
```

Comment:
Sorry I don't understand what you want to say. What do you mean by "call cases"?